base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -871,6 +875,25 @@ impl BlockchainBackend for LMDBDatabase {
 
         Ok(())
     }
+
+    fn get_monero_seed_height(&self, seed: &str) -> Result<u64, ChainStorageError> {
+        let txn = ReadTransaction::new(&*self.env).map_err(|e| ChainStorageError::AccessError(e.to_string()))?;
+        Ok(lmdb_get(&txn, &self.monero_seed_height_db, seed)?.unwrap_or(0))
+    }
+
+    /// This will store the seed key with the height. This is called when a block is accepted into the main chain.
+    /// This will only update the hieght of the seed, if its lower then currently stored.
+    fn update_monero_seed_height(&self, seed: &str, height: u64) -> Result<(), ChainStorageError> {
+        let txn = ReadTransaction::new(&*self.env).map_err(|e| ChainStorageError::AccessError(e.to_string()))?;
```

Comment:
Use the WriteTransaction for both calls here

Reply:
Actually, this should be moved to an operation inside of DbTransaction, so use that WriteTransaction for both